### DISCUSSION TITLE
Development still going?

### DISCUSSION THREAD

**Discussion by Psykehuset** [apohl79/audiogridder#862] [Ideas]
I was hoping to see information about how the development and while i was looking i was wondering how to support the project. I found the donate button on the main page here on git.

So audiogridder is still being developed?
If so i might need to chip in.

Only feature i wish for is that the server window would be isolated and on a virtual monitor. This because one of the servers we run audiogridder on is also used as a workstation. When a audiogridder plug-in is being controlled with gui the person working on the computer can't do anything because mouse and keyboard is remote controlled.

Now i have no idea how to get passed this and thus also allowing multiple users to use the same server at the same time. But that would be incredible.

Considered patreon?

Anyway. Audiogridder has opened up possibilities for clients at my studio to bring any shitty laptop in and get access to lots of processing power and plugins. Before they had to use the studios own computer and DAW. 

Thank you and keep up the great work!

**Comment by apohl79** [comment]
Yes development is in progress, a bit slow these days, but I don't intend to abandon the project.

If you'd like to get involved on the dev side, happy days! You can mail me at audiogridder at e47.org.

**Comment by yoshimodular** [comment]
I'm a very happy AudioGridder user, but now I start having some issues with some plugins. Is there any plan to keep this development active? I would be happy to support this as a paid product too (I have donated).

Thanks!

**Reply by bluetardis** [reply]
Same here

**Reply by ralphonz** [reply]
Me too.  This is well on the way to being an excellent product.

**Comment by paolocampitelli** [comment]
up......

**Comment by house-producer** [comment]
In my opinion, AudioGridder is one of the most important tools ever developed, and I've only ever needed to use it locally.   Thanks for all the effort in putting this awesome tool into existence.  

+1 for continued support!
